### PR TITLE
Adds beer box

### PR DESCRIPTION
**Pull request by IThe-TI**
[content][sprites]

Want to give your colleagues a refreshing beer, but don't want to be that guy with a ton of bottles in his already full backpack?
Fear not! With the latest NT technology, you are now able to fold a beer box with one sheet of cardboard!
beerbox

:cl:
 * rscadd: Added the beer box, craftable with one sheet of cardboard

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -720,3 +720,33 @@
...
+	icon_state = "beerbox0"
+
+/obj/item/weapon/storage/fancy/beer_box/New()
+	..()
+	if(empty)
+		update_icon()
+		return
+	for(var/i = 1; i <= storage_slots; i++)
+		new /obj/item/weapon/reagent_containers/food/drinks/beer(src)
+	return
```
Comment (DamianX) [inline code/game/objects/items/weapons/storage/fancy.dm]: return at the end of the proc does nothing, remove it

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 2]
```diff
@@ -720,3 +720,33 @@
...
+/obj/item/weapon/storage/fancy/beer_box/empty
+	empty = 1
+	icon_state = "beerbox0"
+
+/obj/item/weapon/storage/fancy/beer_box/New()
+	..()
+	if(empty)
+		update_icon()
+		return
+	for(var/i = 1; i <= storage_slots; i++)
```
Comment (DamianX) [inline code/game/objects/items/weapons/storage/fancy.dm]:
```suggestion
	for(var/i in 1 to storage_slots)
```
it's preferred to use this kind of for-loops. they are easier to read and faster (not that it matters here).

**Review by DamianX — APPROVED** [review]
neat

**Comment by Kurfursten** [comment]
Just need box wine and our rednecks will be KITTED

**Comment by PrimeDSS13** [comment]
why do they have a red S on them if it's NT technology

**Comment by IThe-TI** [comment]
> 
> 
> why do they have a red S on them if it's NT technology

The space beer bottles have a red S on them, so I figured a box to carry space beer around would have that logo printed on it

**Comment by beesil** [comment]
ale bottles and cans should fit in it as well, otherwise 👍

**Comment by PrimeDSS13** [comment]
> 
> 
> > why do they have a red S on them if it's NT technology
> 
> The space beer bottles have a red S on them, so I figured a box to carry space beer around would have that logo printed on it

you know I never actually noticed that, fair,

**Pull request merged**